angular.json

```diff
@@ -100,6 +100,7 @@
               }
             ],
             "styles": [
+                "node_modules/angular2-draggable/css/resizable.min.css",
```

Comment:
Does this have to be defined at the root level?

Reply:
We have to add it somewhere to make resizable work. Any other ideas?

Reply:
Added the css file directly to exercise-chatbot file and it is working, would this be okay? @Hialus @undernagruzez 

    styleUrls: ['./exercise-chatbot.component.scss', '../../../../../../node_modules/angular2-draggable/css/resizable.min.css'],

Reply:
Update: It only works in angular.json.